app/views/users/edit.html.erb

```diff
@@ -131,3 +130,11 @@
     </div>
   </main>
 <%end%>
+
+<script>
```

Comment:
Probably worth pulling this out of the view and into a separate js file, just to encourage future contributors to follow that pattern.